hydrosdk/application.py

```diff
@@ -25,7 +27,7 @@ class ApplicationStatus(Enum):
     READY = 2
 
 
-class Application:
+class Application(PredictableApplication):
```

Comment:
Addressed my opinion in `predictor` review.

Reply:
Done